tests/foreman/rhai/test_rhai.py

```diff
@@ -258,14 +221,27 @@ def test_positive_playbook_download():
 
     :expectedresults: sane playbook downloaded
 
-    :CaseAutomation: notautomated
+    :CaseAutomation: automated
 
     :CaseLevel: System
     """
+    plan_name = f"{test_name}_{gen_string('alpha', 6)}"
+    autosession.insightsplan.create(plan_name, ['/etc/dnf/dnf.conf'])
+    file_path = autosession.insightsplan.download_playbook(plan_name)
 
+    with open(file_path) as yamlfile:
+        yaml_content = yaml.load(yamlfile.read(), Loader=yaml.SafeLoader)
```

Comment:
yaml can load from stream directly. It has potential to be more effective as the file might not necessary be loaded completely into the memory.
```suggestion
        yaml_content = yaml.load(yamlfile, Loader=yaml.SafeLoader)
```

Reply:
That file has around 50 lines, so I highly doubt we will see any impact on performance or effectiveness.
Changed.